projects/GKCore/GKCore/Controllers/AssociationEditDlgController.cs

```diff
@@ -41,8 +41,7 @@ public GDMAssociation Association
             set {
                 if (fAssociation != value) {
                     fAssociation = value;
-                    //fTempPerson = fBase.Context.Tree.GetPtrValue(fAssociation);
-                    fTempPerson = fAssociation.Individual;
+                    fTempPerson = fBase.Context.Tree.GetPtrValue(fAssociation);

```

Comment:
I'm not sure why you have not replaced this by yourself

Reply:
There was a nasty bug in the test of this part. I didn't manage (something I don't understand) using NSubstitute functions. When I make a stub from BaseWindow, I needed Returns (realInstanceOfGDMTree) to be executed at the BaseContext level - and this did not.

Reply:
Bug with using NSubstitute in the /projects/GKTests/GKCore/Controllers/AssociationEditControllerTests.cs/Test_AssociationEditDlgController()